src/index.html

```diff
@@ -10,13 +10,53 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    >
+
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Moyo header</h1>
-  </body>
+  <header>
+    <a
+      href="#"
+      class="logo"
+    >
+      <img
+        src="images/logo.png"
+        alt="logo"
+      />
+    </a>
```

Comment:
The 'alt' attribute for images should describe the image content, not just say 'logo'. For example, 'Company Logo' would be more descriptive.